rockylinux-9/Containerfile

```diff
@@ -0,0 +1,46 @@
+FROM rockylinux:9
+
+RUN dnf update -y \
+    && dnf install -y --allowerasing \
+      coreutils \
+      cpio \
+      dhclient \
+      e2fsprogs \
+      ethtool \
+      findutils \
+      initscripts \
+      ipmitool \
+      iproute \
+      kernel-core \
+      kernel-modules \
+      ncurses \
+      net-tools \
+      NetworkManager \
+      nfs-utils \
+      openssh-clients \
+      openssh-server \
+      pciutils \
+      policycoreutils-python-utils \
+      psmisc \
+      rsync \
+      rsyslog \
+      strace \
+      selinux-policy-targeted \
+      wget \
+      which \
+      words \
+      rdma-core \
+    && dnf clean all
+
+RUN touch /etc/sysconfig/disable-deprecation-warnings
+
+RUN semanage fcontext -N -a -t bin_t /warewulf/wwclient
```

Comment:
Can you add a comment to this line that indicates what breaks without it?

Reply:
In my testing with WW4 built from source on a Rocky 9.1host, the `wwclient` binary that is dropped onto a freshly booted node with the `wwinit` overlay didn't have an appropriate SELinux label for `/warewulf/wwclient` so the `wwclient` service failed to start when SELinux was otherwise setup correctly.

Could that possibly just be an issue with my WW4 source build?

Reply:
(this line being the `semanage fcontext` line.)

Reply:
I don't know how the overlay system handles selinux contexts; but I'm not surprised that you needed to add this; I'm just thinking it's not immediately obvious; so justputting a note about what you said here in the file as a comment would be useful for future maintainers.

Reply:
Comment added to explain the reasoning behind `semanage fcontext` for future maintainer sanity.